Book-A-Desk.Domain/CommandHandler.Reservations.fs

```diff
@@ -21,14 +22,17 @@ module BookADeskCommandHandler =
                 |> List.singleton
                 |> Map.ofList
                 |> eventStore.AppendEvents
-                |> Result.map (fun _ ->  events |> List.map ReservationEvent)
+                Result.Ok ()
```

Comment:
We don't seem to use the response for this handler and the events have changed a little bit, so I changed it to unit instead of the domain events.